manifests/a/AIMP/AIMP/v5.03.2394, 30.06.2022/AIMP.AIMP.yaml

```
@@ -0,0 +1,8 @@
+# Created with YamlCreate.ps1 v2.1.2 $debug=QUSU.5-1-19041-1682
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.1.0.schema.json
+
+PackageIdentifier: AIMP.AIMP
+PackageVersion: v5.03.2394, 30.06.2022
```

```suggestion
PackageVersion: 5.03.2394
```
Looks like they changed their versioning scheme so this must match Control Panel.